src/pages/staking/index.js

```diff
@@ -1,422 +1,668 @@
-  color: ${(props) =>
-    props.isActive ? props.theme.colors.primary : props.theme.colors.text};
-  box-shadow: ${(props) =>
-    props.isActive ? props.theme.colors.tableBoxShadow : `none`};
+const ColorH3 = styled.h3`
+  grid-area: ${({ color }) => {
+    switch (color) {
+      case "stakingGold":
+        return "solo-title"
+      case "stakingGreen":
+        return "saas-title"
+      case "stakingBlue":
+        return "pool-title"
+      default:
+        return ""
+    }
+  }};
+  color: ${({ theme, color }) => theme.colors[color]};
+`
+
+const ButtonContaier = styled.div`
   display: flex;
-  align-items: center;
-  padding: 1rem 1.5rem;
-  margin: 0.5rem;
-  cursor: pointer;
+  gap: 1rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+  }
 `
 
-const OptionText = styled.div`
-  font-size: 1.5rem;
-  line-height: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    font-size: 1rem;
-    font-weight: 600;
+const StyledButtonLink = styled(ButtonLink)`
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: 100%;
   }
 `
 
-const StakeContainer = styled.div`
-  margin: 0 auto;
-  max-width: ${(props) => props.theme.breakpoints.m};
-  display: flex;
-  flex-direction: column;
-  text-align: center;
+const StyledCard = styled(Card)`
+  justify-content: flex-start;
+  h3 {
+    font-weight: 700;
+    margin: 0 0 1rem;
+  }
 `
 
-const paths = [
+const benefits = [
   {
-    emoji: ":money_with_wings:",
-    title: <Translation id="page-staking-title-1" />,
-    description: <Translation id="page-staking-desc-1" />,
+    title: "Earn rewards",
+    emoji: "💰",
+    description:
+      "Rewards are given for actions that help the network reach consensus. You'll get rewards for batching transactions into a new block or checking the work of other validators because that's what keeps the chain running securely.",
   },
   {
-    emoji: ":warning:",
-    title: <Translation id="page-staking-title-2" />,
-    description: <Translation id="page-staking-desc-2" />,
+    title: "Better security",
+    emoji: ":shield:",
+    description:
+      "The network gets stronger against attacks as more ETH is staked, as it then requires more ETH to control a majority of the network. To become a threat, you would need to hold the majority of validators, which means you'd need to control the majority of ETH in the system–that's a lot!",
   },
   {
-    emoji: ":clipboard:",
-    title: <Translation id="page-staking-title-3" />,
-    description: <Translation id="page-staking-desc-3" />,
-    url: "/developers/docs/apis/backend/#available-libraries",
-    link: <Translation id="page-staking-link-1" />,
+    title: "More sustainable",
+    emoji: "🍃",
+    description:
+      "Stakers don't need energy-intensive computers in order to participate in a proof-of-stake system–just a home computer or smartphone. This will make Ethereum better for the environment.",
```

Comment:
I think we should follow this section with a link to https://ethereum.org/en/energy-consumption/ e.g. "More on Ethereum's energy consumption".